containers/unit-tests/README.md

```diff
@@ -11,36 +11,51 @@ will re-use an already existing `node_modules/` directory.
 
 ## Building
 
-In a built cockpit tree you can run
-
-    $ make unit-tests-container
-
-which will build the `cockpit/unit-tests` and `cockpit/unit-tests:i386`
-containers.
+The `build` script will build the `cockpit/unit-tests` and
+`cockpit/unit-tests:i386` containers.  It should be run as root.
 
 ## Running tests
 
-Tests in that container for the default configuration get started with
+Tests in that container get started with the `start` script.  By default, this
+script runs the unit tests on amd64.  The script accepts a number of arguments
+to modify its behaviour:
 
-    $ make unit-tests-container-run
+ - `CC=othercc` to set the `CC` environment variable inside the container (ie:
+   to build with a different compiler)
+ - `:tag` to specify a different tag to use for the `cockpit/unit-tests` image
+   (eg: `i386`)
+ - `shell` to specify that an interactive shell should be launched instead of
+   running the unit tests
 
-or equivalently with
+Some examples:
 
-    $ sudo docker run --shm-size=512M -ti --volume `pwd`:/source:ro cockpit/unit-tests
+    $ sudo ./start           # run the unit tests on amd64
 
-You can pass `--env=CC=clang` to build with Clang instead of gcc, or run
-`cockpit/unit-tests:i386` to run on a 32 bit architecture.
+    $ sudo ./start CC=clang  # run the unit tests on amd64, compiled with clang
```

Comment:
Note to self: try the container with unpriv podman and for that drop the sudo here. (follow-up matter)